.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,38 +1,46 @@
----
-name: Bug report
-about: Create a report to help us improve
-title: ''
-labels: ''
-assignees: ''
 
----
+**Please only open an issue here, when advised by a moderator in OMV Forum**
```

Comment:
```suggestion
**Please only open an issue here, when advised by a moderator in openmediavault forum**
```
I think we should be more specific here, otherwise users will reach out to the oil company forum :-)